plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx

```diff
@@ -76,77 +85,68 @@ export const GroupProfileCard = ({
     ?.map(groupItem => groupItem.target.name)
     .toString();
 
-  const childrens = group?.relations
+  const childRelations = group?.relations
     ?.filter(r => r.type === RELATION_PARENT_OF)
     ?.map(groupItem => groupItem.target.name);
```

Comment:
Btw, we now have a helper for this: `getEntityRelations` in `@backstage/plugin-catalog-react`

Reply:
Awesome - thanks! I gave it a shot, but could use your review to ensure I used it correctly. 🤜🤛

Reply:
Looks good